File: 2310817210014_Damarjati Suryo Laksono/Modul 2/C/PRAK204-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    float pi = 3.142857;
    double jari, tinggi;
    scanf("%lf %lf", &jari, &tinggi);
    double volume = pi * jari * jari * tinggi;
    double luas = 2 * pi * jari * (jari + tinggi);
    double keliling = 2 * pi * jari;
    printf("Volume   = %.2f\n", volume);
    printf("Luas     = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);
    return 0;
}

File: 2310817210014_Damarjati Suryo Laksono/Modul 2/Python/PRAK204-2310817210014-DamarjatiSuryoLaksono.py
v = input ().split(" ")

if len(v) == 1:
    jari = float(v[0])
    tinggi = float(input())
    
elif len(v) == 2:
    jari, tinggi = map(float, v)

pi = 22/7
volume = round(pi*jari*jari*tinggi,2)
luas = round(2*pi*jari*jari+tinggi,2)
keliling = round(2*pi*jari,2)
print ("Volume =",volume)
print ("Luas =" ,luas)
print ("Keliling =", keliling)